ref/general/microshed.adoc

```diff
@@ -6,37 +6,35 @@
 
 == Why integration testing
 
 You need to be sure that your application works as well in production as it does in the development environment.
 Integration tests assess multiple test classes and components, but take longer and can be complicated to set up and configure.
-With shorter development cycles and limited time available, developers often run unit tests because they are easier and faster.
-MicroShed Testing framework streamlines your integration tests with the Testcontainers library for a better workflow.
+Altenately, unit tests involve testing individual modules of an application and are shorter and less complicated to run.
+With shorter development cycles and limited time available, developers often run unit tests.
+MicroShed Testing streamlines your integration tests with the Testcontainers framework for an efficient workflow.
 
 == Development-production parity
 
-Development-production parity, or dev/prod parity, is one of the factors in the link:https://12factor.net/[twelve-factor app], a methodology to build modern applications.
+Development-production parity, or dev/prod parity, is one of the factors in the link:https://12factor.net/ [twelve-factor app], a methodology to build modern applications.
 The idea behind dev/prod parity is to keep the development, staging, and production environments as similar as possible in terms of time, personnel, and tools.
 To simplify the development process, developers often use different tools in development than they do in production.
 For example, you might use a local Maven build in development, but the application might be deployed to a Docker container in production.
 Differences between the environments can cause a test to fail in production, even though it passed in development.
-The application might work on your computer but not in production.
 So it is important to use the same services in development as in production.
 MicroShed Testing framework helps achieve dev/prod parity because it tests the application by using the same environment that is used in production.
 
-== Using MicroShed Testing
+== Writing integration tests with Microshed Testing
 
-You can easily deploy your microservices in different environments by using containers.
-Containers are convenient and combine everything that you need to run a service into a single unit.
-They are lightweight, disposable, configurable, and manageable with an orchestration software like Kubernetes.
-You can leverage containers in a test environment by using the Testcontainers framework, which is an open source library that supports JUnit tests.
+You can deploy your microservices in different environments by using containers.
+With the Testcontainers framework, which is an open source library that supports JUnit tests, you can use containers in a test environment.
+Microshed implements testcontainers to support dev/prod parity in testing.
```

Comment:
Microshed implements testcontainers to support dev/prod parity in testing.

Should this be "Microshed Testing" ?and should Testcontainers be capitalized?

I think if you reversed the order of sentences on lines 37 and 38 it might flow better. That way you would introduce microshed before explaining it's relation to testcontainers